apps/condo/domains/common/hooks/useCurrentBuild.tsx

```diff
@@ -0,0 +1,21 @@
+import useSWR from 'swr'
```

Comment:
Why not let user to reload page by himself? He might be in the middle of Ticket editing...

Reply:
Also, why do we need to have swr now? We already have fetch as HTTP client...

Reply:
swr used for cache storing. I'll add "TRICK DESCRIPTION" section 😄